docs/monitoring/metrics.md

```diff
@@ -329,11 +329,11 @@ or by assigning unique names to jobs and operators.
 Metrics can be exposed to an external system by configuring one or several reporters in `conf/flink-conf.yaml`. These
 reporters will be instantiated on each job and task manager when they are started.
 
-- `metrics.reporters`: The list of named reporters.
 - `metrics.reporter.<name>.<config>`: Generic setting `<config>` for the reporter named `<name>`.
 - `metrics.reporter.<name>.class`: The reporter class to use for the reporter named `<name>`.
 - `metrics.reporter.<name>.interval`: The reporter interval to use for the reporter named `<name>`.
 - `metrics.reporter.<name>.scope.delimiter`: The delimiter to use for the identifier (default value use `metrics.scope.delimiter`) for the reporter named `<name>`.
+- `metrics.reporters`: (optional) An include list for reporters to instantiate. By default all configured reporters will be used.
```

Comment:
"An include list for reporters" -> "A comma-separated list of reporter names"? Not sure if we need to specify "comma-separated".

Reply:
I guess it can't hurt to explicitly mention "comma-separated", but I don't have a strong preference, we have an example after all.